test/common/testlib.py

```diff
@@ -2097,6 +2097,15 @@ def write_file(self, path: str, content: str, append: bool = False, owner: Optio
         m = self.machine
         self.restore_file(path, post_restore_action=post_restore_action)
         m.write(path, content, append=append, owner=owner, perm=perm)
+        
+    
+    def reboot(self, timeout_sec):
```

Comment:
There is no default value for `timeout_sec` so you can't use it as `self.reboot`. `machine.reboot` comes from cockpit/bots's `machine_core/ssh_connection.py` which has a default value of 180 seconds we want to retain.

Reply:
`def reboot(self, timeout_sec=180)` so 180 sec should I add to the timeout_sec as a default value
@jelly

Reply:
We could do that but we also could leave it defined in bots so doing something like:
```
def reboot(self, timeout_sec=None)
    if timeout_sec is None:
        machine.reboot()
    else:
        machine.reboot(timeout_sec)
```

Adding typing might be nice as well.